build-files/config.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
Everything in **build-files/config.json** is meant to be used for local development. Should be used as is.
Also, everything in this file can be overridden via **build-files/secrets.json**